static/app/views/settings/components/teamSelect.tsx

```diff
@@ -38,6 +35,10 @@ type Props = {
    * Prevent changes to a SCIM-provisioned member
    */
   enforceIdpProvisioned: boolean;
+  /**
+   * The OrganizationMember that we are acting on
+   */
+  member: Member;
```

Comment:
does this need to be optional because this component is also used to assign a project to a team?